moon/skyline.py

```diff
@@ -0,0 +1,100 @@
+"""
+  Moon Skyline analysis
+"""
+import math
+
+import numpy as np
+import cv2  # for debug
+
+
+def skyline(img):
```

Comment:
How does this interact with noise in the data from camera?

Reply:
badly :(. I wanted to update the blog, which I did not, so here are two examples:
![skyline-noise](https://user-images.githubusercontent.com/5664353/82599430-8ec24d00-9bac-11ea-8dc6-f01382631f8a.jpg)
![skyline-noise2](https://user-images.githubusercontent.com/5664353/82599436-93870100-9bac-11ea-8a6b-be301033165d.jpg)

... the idea (not implemented yet) was to detect "noise" and ignore the frame for ver0.

Reply:
I was going to suggest - since the noisy frames are isolated and occur with limited frequency, a simple detection and skipping over such a frame may be a better way forward than trying to include noisy images in the analysis.